Description: "use encoding" is removed in perl 5.26
 As "use utf8" for the source is already there, just make sure
 we open the standard file handles as UTF-8.
Origin: vendor
Bug-Debian: https://bugs.debian.org/871349
Forwarded: no
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2017-08-08

--- a/bin/verb_reverse.pl
+++ b/bin/verb_reverse.pl
@@ -9,7 +9,7 @@
 use strict;
 use locale;
 use utf8;
-use encoding 'utf8';
+use open qw(:std :encoding(UTF-8));
 
 
 my $UK_CAP	="'АБВГҐДЕЄЖЗИІЇЙКЛМНОПРСТУФХЦЧШЩЬЮЯ";
--- a/bin/affconv.pl
+++ b/bin/affconv.pl
@@ -5,7 +5,7 @@
 use strict;
 use locale;
 use utf8;
-use encoding 'utf8';
+use open qw(:std :encoding(UTF-8));
 
 my (%pfx, %sfx);
 

